### ISSUE TITLE
Diverge with augmented data

### ISSUE THREAD

**Issue by a00achild1** [baidu-research/ba-dls-deepspeech#17]
I'm trying to train a model with augmented LibriSpeech train-other-500 dataset.
The speech was augmented by adding noise and changing the tempo.
But the loss couldn't converge.
I have tried letting the learning rate small(1e-5), adding more GRU units, or adding more nodes, but none of these works.
Is anyone has a solution or suggestions?
![plot_7rnn1200_libri500_aug](https://cloud.githubusercontent.com/assets/20927157/22854010/bd4edc72-f09e-11e6-84f8-812d7385cf1b.png)
